openapi_python_client/openapi_parser/openapi.py

```diff
@@ -236,14 +236,16 @@ def _iterate_properties() -> Generator[Property, None, None]:
 
     @staticmethod
     def from_dict(d: Dict[str, Dict[str, Any]], /) -> OpenAPI:
-        """ Create an OpenAPI from dict """
+        """ Create an OpenAPI from dict 
+        :rtype: object
```

Comment:
Why is this necessary?  Does mypy have a problem with `-> OpenAPI` for some reason?